utilities_common/multi_asic.py

```diff
@@ -114,6 +114,17 @@ def multi_asic_display_default_option():
 ]
 
 def multi_asic_namespace_validation_callback(ctx, param, value):
+    # TODO: this condition has a bug: it should be either "multi_asic.is_multi_asic()"
+    # or "self.is_multi_asic", but changing it to correct version will lead to the 
+    # following cli not working on single-asic:
+    # show ip(v6) bgp neighbors/network, show storm_control add/del
+    # e.g. 
+    # admin@str2-7050cx3-acs-02:/usr$ show ip bgp neighbor -n asic0
+    # -n/--namespace is not available for single asic
+    # Aborted!
+    # admin@str2-7050cx3-acs-02:/usr$ show ip bgp neighbor
+    # -n/--namespace is not available for single asic
```

Comment:
are you fixing this in another PR? Can you open a github issue iso adding comment in the code

Reply:
raised issue: https://github.com/sonic-net/sonic-buildimage/issues/13560